python/tskit/trees.py

```diff
@@ -59,6 +59,8 @@
 
 BaseInterval = collections.namedtuple("BaseInterval", ["left", "right"])
 
+EdgeDiffs = collections.namedtuple("EdgeDiffs", ["interval", "edges_out", "edges_in"])
```

Comment:
I think this is probably better as singular, ``EdgeDiff``. Each thing we return is a single diff, and together they are the diffs.